3.0/README.md

```diff
@@ -0,0 +1,171 @@
+
+## 3.1 `attribution`
+
+OPTIONAL. Default: null.
+
+Contains an attribution to be displayed when the map is shown to a user. Implementations MAY decide to treat this as HTML or literal text. For security reasons, make absolutely sure that this field can't be abused as a vector for XSS or beacon tracking.
+
+```JSON
+{
+  "attribution": "<a href='http://openstreetmap.org'>OSM contributors</a>"
+}
+```
+
+## 3.2 `bounds`
+
+OPTIONAL. Default: [-180, -90, 180, 90].
+
+The maximum extent of available map tiles. Bounds MUST define an area covered by all zoom levels. The bounds are represented in WGS:84 latitude and longitude values, in the order left, bottom, right, top. Values may be integers or floating point numbers. The minimum/maximum values for longitude and latitude are -180/180 and -90/90 respectively. Bounds MUST NOT "wrap" around the ante-meridian. If bounds are not present, the default value MAY assume the set of tiles is globally distributed.
+
+```JSON
+{
+  "bounds": [ -180, -85.05112877980659, 180, 85.0511287798066 ]
+}
+```
+
+## 3.3 `center`
+## 3.4 `data`
+## 3.5 `description`
+
+OPTIONAL. Default: null.
+
+A text description of the tileset. The description can contain any legal character. Implementations SHOULD NOT interpret the description as HTML.
+
+```JSON
+{
+  "description": "We are volcanoes. When we women offer our experience as our truth, as human truth, all the maps change. There are new mountains."
+}
+```
+
+## 3.6 `grids`
+## 3.7 `legend`
+
+## 3.8 `maxzoom`
+
+OPTIONAL. Default: 30. >= 0, <= 30.
+
+An integer specifying the maximum zoom level. MUST be >= minzoom.
+
+```JSON
+{
+  "maxzoom": "11"
+}
+```
+
+## 3.9 `minzoom`
+
+OPTIONAL. Default: 0. >= 0, <= 30.
+
+An integer specifying the minimum zoom level. MUST be <= maxzoom
+
+```JSON
+{
+  "minzoom": "0"
+}
+```
+
+## 3.10 `name`
+
+OPTIONAL. Default: null.
+
+A name describing the tileset. The name can contain any legal character. Implementations SHOULD NOT interpret the name as HTML.
+
+```JSON
+{
+  "name": "Earthsea v2"
+}
+```
+
+## 3.11 `scheme`
+
+OPTIONAL. Default: "xyz". 
+
+Either "xyz" or "tms". Influences the y direction of the tile coordinates. The global-mercator (aka Spherical Mercator) profile is assumed.
+
+```JSON
+{
+  "scheme": "xyz"
+}
+```
+
+## 3.12 `template`
+## 3.13 `tilejson`
+
+REQUIRED.
+
+A semver.org style version number. Describes the version of the TileJSON spec that is implemented by this JSON object.
+
+```JSON
+{
+  "tilejson": "3.0"
+}
+```
+
+## 3.14 `tiles`
+
+REQUIRED. 
+
+An array of tile endpoints. {z}, {x} and {y}, if present, are replaced with the corresponding integers. If multiple endpoints are specified, clients may use any combination of endpoints. All endpoints MUST return the same content for the same URL. The array MUST contain at least one endpoint. The tile extension is NOT limited to any particular format. Some of the more popular are: mvt, vector.pbf, png, webp, and jpg.
```

Comment:
All endpoints returning the same content for the same URL is a problem, thanks to caching, different servers, etc. At various times this would have made it impossible to provide a tilejson for [a-c].tile.osm.org, since each could have the DNS resolve to a different server.